app/helpers/oregon_digital/facet_helper_behavior.rb

```diff
@@ -0,0 +1,13 @@
+module OregonDigital::FacetHelperBehavior
+  def render_facet_item(facet_field, item)
```

Comment:
Can you add an `OVERRIDE FROM BLACKLIGHT` comment here? So we know what to look back on if this method changes

Reply:
👍